tests/Catalog/CatalogServiceTest.php

```diff
@@ -848,16 +865,21 @@ public function testGetCategoryTree()
         $this->assertSame('', $category->getDescription());
         $this->assertSame(10, $category->getPosition());
         $this->assertSame(0, $category->getProductCount());
+        $this->assertSame([['id' => 2, 'name' => 'Catégorie principale', 'slug' => 'categorie-principale']], $category->getCategoryPath());
```

Comment:
on préfère éviter de renvoyer des tableaux comme ca, quand les champs sont bien définis on préfère les objets

Reply:
Noté, je vais revoir !